image/cli/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM ubuntu
```

Comment:
Terrific question!

Head over here and read up on everything (yes, it's long).  I comment somewhere in the middle.  Then let's chat about it: https://github.com/docker/docker/issues/6396

tldr: We do this so that the cli container doesn't get stinkered by github (and others) rate limits for anonymous access.  Also allows containers to access private repositories.